components/user_settings/sidebar/user_settings_sidebar.jsx

```diff
@@ -733,9 +733,8 @@ export default class UserSettingsSidebar extends React.Component {
                     </h3>
                     <div className='divider-dark first'/>
                     {channelOrganizationSection}
-                    {showChannelOrganization && <div className='divider-light'/>}
                     {channelSwitcherSection}
-                    {showUnusedOption && <div className='divider-light'/>}
+                    {showUnusedOption ? <div className='divider-light'/> : <div className='divider-dark'/>}
```

Comment:
I wonder if it would be easier to do this in CSS instead. 0/5

Reply:
Yeah, I think it make sense.  Maybe it's easier to implement if we don't have to specifically assign dark or light divider. It's hard to differentiate between the two by just looking at it.  I'll start a discussion at UX channel.